### DISCUSSION TITLE
navicat查看数据表报错，doris版本是2.0.11，navicat是15，mysql是8.0.34

### DISCUSSION THREAD

**Discussion by liz-201** [apache/doris#36780] [A - General / Q&A]
**这是我的建表语句：**
CREATE TABLE IF NOT EXISTS gov_log_login(
    id BIGINT COMMENT "编号",
    create_time datetime COMMENT "创建时间",
    success TINYINT COMMENT "接口是否返回成功 0：失败 1：成功",
    tenant_id VARCHAR ( 128 ) COMMENT "租户ID",
    service_id VARCHAR ( 32 ) COMMENT "服务ID",
    server_host VARCHAR ( 255 ) COMMENT "服务器名",
    server_ip VARCHAR ( 255 ) COMMENT "服务器IP地址",
    env VARCHAR ( 255 ) COMMENT "服务器环境",
    type CHAR ( 1 ) COMMENT "日志类型",
    title VARCHAR ( 255 ) COMMENT "日志标题",
    method VARCHAR ( 10 ) COMMENT "操作方式",
    request_uri VARCHAR ( 255 ) COMMENT "请求URI",
    user_agent VARCHAR ( 10 ) COMMENT "用户代理",
    remote_ip VARCHAR ( 255 ) COMMENT "操作IP地址",
    method_class VARCHAR ( 255 ) COMMENT "方法类",
    method_name VARCHAR ( 255 ) COMMENT "方法名",
    params VARCHAR ( 65530 ) COMMENT "操作提交的数据",
    time VARCHAR ( 64 ) COMMENT "执行时间",
    create_by VARCHAR ( 64 ) COMMENT "创建者",
    app_zone_id VARCHAR ( 64 ) COMMENT "应用空间id",
    user_id BIGINT COMMENT "用户id",
    result STRING COMMENT "接口返回参数"
) ENGINE = OLAP 
DUPLICATE KEY (id, create_time, success) 
COMMENT "接口日志表" 
PARTITION BY RANGE (create_time) () 
DISTRIBUTED BY HASH (id) BUCKETS 1 
PROPERTIES (
    "dynamic_partition.enable" = "true",
    "dynamic_partition.time_unit" = "MONTH",
    "dynamic_partition.prefix" = "p",
    "dynamic_partition.buckets" = "2",
    "dynamic_partition.end" = "1",
    "dynamic_partition.replication_num" = "1"
);
**在navicat中执行后并无报错，能成功建立数据表，但是在展示数据表和表结构的时候报错，截图如下：**
![image](https://github.com/apache/doris/assets/66368352/1b4616f6-f5f8-44c5-b55c-fbb2360c6179)

**查看doris的log，发现信息如下：**
![image](https://github.com/apache/doris/assets/66368352/604bb4fe-48ff-4a6b-a9e4-3bd4aa899392)

**Comment by ixzc** [comment]
中文问题，建议去ask.selectdb.com去问问题
